dist/radar_client.js

```diff
@@ -381,6 +381,16 @@ Client.prototype._createManager = function() {
   });
 
   manager.on('activate', function() {
+    var socket = client._socket;
+    if (!client.name) {
+      client.name = socket.id;
```

Comment:
Yes, socket id changes on each reconnect; client.name remains at the same value as long as it is set, and the client.name/client.id pair keeps track of this association as long as the client is not completely closed.  There is more information here: https://zendesk.atlassian.net/wiki/display/INFR/Saving+Client+State+on+the+Server